sdks/java/io/hadoop/input-format/src/main/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIO.java

```diff
@@ -389,6 +394,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
     private long boundedSourceEstimatedSize = 0;
```

Comment:
for populateDisplayData (next function up, but I can't comment there): when I run this with the elasticsearch IO IT on dataflow, I don't see any of the display data about the hadoop configuration populated. 

looking at:
https://github.com/diptikul/incubator-beam/blob/5c90e14ed6bdbfa2f486b85dc9ea5a68d755d9ad/sdks/java/io/hadoop/jdk1.8-tests/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/integration/tests/HIFIOElasticIT.java#L194

I would expect to see items about es.internal.es.version, etc... and I'm not seeing them in the dataflow UI 


Given that I believe there are a number of hadoop configuration items in that job, we should be seeing data there. I thought it might be a dataflow display problem, but when I look at what we send the UI, send other display data, but not that one. 

Can you make sure the appropriate display data is getting populated?